pycbc/filter/qtransform.py

```diff
@@ -42,7 +42,160 @@
 from pycbc.types import zeros
 from numpy import fft as npfft
 import os
+import h5py
+from pycbc.filter import highpass_fir, matched_filter
+from pycbc.psd import welch, interpolate
 
+def pycbc_insp_main(segments, filename='q_info.hdf'):
```

Comment:
Please give things a bit more meaningful names, not tied to a specific executable. Presumably this function could be used elsewhere.